scene/main/node.cpp

```diff
@@ -3088,6 +3088,22 @@ void Node::set_display_folded(bool p_folded) {
 	data.display_folded = p_folded;
 }
 
+void Node::set_display_folded_bind(bool p_folded) {
+	ERR_THREAD_GUARD
+
+	if (p_folded == data.display_folded) {
+		return;
+	}
+	set_display_folded(p_folded);
+
+#ifdef TOOLS_ENABLED
+	if (fold_changed_callback.is_valid() && is_part_of_edited_scene()) {
+		Array args;
+		fold_changed_callback.callv(args);
```

Comment:
I wish #79341 was merged :/

Reply:
Then you should lobby core reviewers for a review :P

Reply:
Then you should lobby core reviewers for a review :P